src/cljs/org/broadinstitute/firecloud_ui/page/method_config_importer.cljs

```diff
@@ -230,15 +230,18 @@
    (fn [{:keys [props state]}]
      [table/Table
       {:columns [{:header "Type" :starting-width 100}
-                 {:header "Namespace" :starting-width 150}
-                 {:header "Name" :starting-width 200 :as-text #(% "name") :sort-by :text
+                 {:header "Name" :starting-width 450 :as-text #(% "name")
```

Comment:
Re: "We need to come up with a better header than "Name"....."  Either entity or item I think is okay, but what about "Method or Configuration" because those describe exactly the item.  Or what about "Method or Configuration Name"?